create-neon/src/bin/create-neon.ts

```diff
@@ -29,31 +30,60 @@ function inferVersions(): Versions {
   };
 }
 
+function deleteNeonDir(dir:string): Promise<void> {
```

Comment:
Nit, please leave a space after the `:` and before the type. This is repeated a few times in the PR.

Hopefully we can get `prettier` configured at some point!